lib/rubocop/cop/layout/first_method_argument_line_break.rb

```diff
@@ -63,13 +63,29 @@ module Layout
       #     }
       #   )
       #
+      # @example AllowedMethods: ['method']
```

Comment:
That's just my two cents. Since Ruby has the `method` method, an example like `some_method` might be preferred.

Reply:
Sure, I just maintained the pattern from the previous examples. Should I update the other examples or just the new ones?

Reply:
I see. The new examples will likely be sufficient.

Reply:
Updated!